blueflood-http/src/main/java/com/rackspacecloud/blueflood/tracker/Tracker.java

```diff
@@ -123,6 +127,16 @@ public static void track(HttpRequest request) {
         }
     }
 
+    static String findTid( String uri ) {
+
+        Matcher m = patternGetTid.matcher( uri );
+
+        if( m.matches() )
+            return m.group( 1 );
+        else
+            return null;
```

Comment:
Maps can have null keys, for some goofy reason, so Map.contains( null ) won't throw and exception.

I'm fine changing it if its for the best, just didn't want to change behavior if not necessary.